### ISSUE TITLE
Update to Circos 0.69.8+galaxy7 at usegalaxy.org

### ISSUE THREAD

**Issue by jennaj** [galaxyproject/usegalaxy-playbook#313]
The complete package (multiple tools in the suite).

Two primary reasons:

1. The older version has fewer features and is being reported as problematic by users. Jen cannot get it to work either unless the input is very very small, and even then not 100% of the time. Not enough memory allocated?
1. The new version is a big upgrade and is what usegalaxy.eu is using. 

Probably should remove the old version entirely. Q: @hexylena yes or no for removing the old version in your opinion? Also, is there a minimum memory allocation that you would suggest for the new version?

cc @tnabtaf

**Comment by hexylena** [comment]
Yes for removing the old one. It was too buggy, the new one is the one in the paper/works much better.
